indico/web/client/styles/modules/_category.scss

```
@@ -536,6 +536,10 @@ ul.category-resource-qtip {
         border-top: 1px dotted #aaa;
       }
 
+      & .title .ui.label {
```

```suggestion
      .title .ui.label {
```

`& foo` is typically redundant in SCSS